resources/zib/zib-HelpFromOthers.xml

```diff
@@ -0,0 +1,164 @@
+      </binding>
+      <mapping>
+        <identity value="zib-helpfromothers-v3.01-2020EN" />
+        <map value="NL-CM:3.2.9" />
+        <comment value="TypeOfHelp" />
+      </mapping>
+    </element>
+    <element id="CarePlan.activity.detail.scheduled[x]">
+      <path value="CarePlan.activity.detail.scheduled[x]" />
+      <slicing>
+        <discriminator>
+          <type value="type" />
+          <path value="$this" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="CarePlan.activity.detail.scheduled[x]:scheduledString">
+      <path value="CarePlan.activity.detail.scheduled[x]" />
+      <sliceName value="scheduledString" />
+      <short value="Frequency" />
+      <definition value="The frequency of the provided support, e.g. daily/weekly." />
+      <alias value="Frequentie" />
+      <type>
+        <code value="string" />
+      </type>
+      <mapping>
+        <identity value="zib-helpfromothers-v3.01-2020EN" />
+        <map value="NL-CM:3.2.2" />
+        <comment value="Frequency" />
+      </mapping>
+    </element>
+    <element id="CarePlan.activity.detail.performer">
+      <path value="CarePlan.activity.detail.performer" />
+      <short value="CareAid" />
+      <definition value="An CareAid can be a health professional such as nurses, carers and helpers but also a caregiver or healthcare provider such as a home care organization. Either a reference to details of the health professional involved in the patient’s care, the details of the caregiver involved in the patient’s support or the details of the organization the health professional/careaid works for in the event that it is not entered as a person." />
+      <alias value="Hulpverlener" />
+      <alias value="Zorgverlener" />
+      <alias value="Mantelzorger" />
+      <alias value="Zorgaanbieder" />
+      <max value="1" />
+      <type>
+        <code value="Reference" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/Practitioner" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/PractitionerRole" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/Organization" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/RelatedPerson" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/Patient" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/CareTeam" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/HealthcareService" />
+        <targetProfile value="http://hl7.org/fhir/StructureDefinition/Device" />
+        <targetProfile value="http://nictiz.nl/fhir/StructureDefinition/zib-HealthProfessional-Practitioner" />
+        <targetProfile value="http://nictiz.nl/fhir/StructureDefinition/zib-ContactPerson" />
+        <targetProfile value="http://nictiz.nl/fhir/StructureDefinition/zib-HealthcareProvider-Organization" />
```

Comment:
It makes more sense to refer to zib-HealthcareProvider (profile on Location) here, as it is the focal profile of the zib HealthcareProvider.

Reply:
Fix

Reply:
This is not possible because location is not supported in the target profiles